fastqheat/config.py

```diff
@@ -0,0 +1,21 @@
+from dataclasses import dataclass
+from functools import cache
+from typing import Optional
+
+
+@dataclass
+class Config:
+    max_retries: int = 1
+
+
+_config = Config()
+
+
+def set_settings(*, max_retries: Optional[int] = None) -> None:
+    if max_retries is not None:
+        _config.max_retries = max_retries
+
+
+@cache
```

Comment:
Why?

Reply:
Caching. I guess I'll need to remove it ;)

Reply:
I am glad I could help